test/db/cmd/cmd_help

```diff
@@ -214,6 +214,177 @@ trits   0t2
 EOF
 RUN
 
+NAME="? 2**3"
+FILE=--
+CMDS=? 2**3
+EXPECT=<<EOF
+int32   8
+uint32  8
+hex     0x8
+octal   010
+unit    8
+segment 0000:0008
+string  "\b"
+fvalue  8.0
+float   0.000000f
+double  0.000000
+binary  0b00001000
+trits   0t22
+EOF
+RUN
+
+NAME="? 10**-2"
+FILE=--
+CMDS=? 10**-2
+EXPECT=<<EOF
+int32   0
+uint32  0
+hex     0x0
+octal   00
+unit    0
+segment 0000:0000
+string  "\0"
+fvalue  0.0
```

Comment:
this is wrong. Please review the code for negative exponents.

Reply:
This seems to be an issue concerning the overall code and how it handles floating point. So I am marking as resolved for now.